doc/configuration.rst

```diff
@@ -0,0 +1,173 @@
+
+- **process owner**:
+    - *can start destruction*: no
+    - *can review destruction*: yes
+    - *can view case details*: yes
+
+- **archivars**:
+    - *can start destruction*: no
+    - *can review destruction*: yes
+    - *can view case details*: no
+
+You can create as many roles as you want and name them as you see fit.
+
+Accounts
+--------
+
+Via **Admin > Authentication and authorization > Users** you can manage individual users
+known to the system. You can perform administrative actions such as:
+
+- assigning a role to a user
+- filling out their name/e-mail address
+- (re)setting their password
+
+ADFS
+----
+
+The RMA admin interface and frontend support logging in through ADFS-backed single sign
+on (SSO).
+
+ADFS 2012 and 2016, and Azure AD are supported. See the `ADFS config guides`_ for
+documentation on how to configure ADFS itself.
+
+The ADFS configuration can be found under **Admin > Configuration > ADFS Configuration**.
+
+Services
+========
+
+The RMA does not store, synchronize or copy case data. All data is retrieved through the
+Zaken, Catalogi and Documenten API. As such, these services need to be configured.
+
+Navigate to **Admin > Configuration > Service** and add the details for your
+environment.
+
+.. note:: RMA supports multiple services of the same type.
+
+Catalogi API
+------------
+
+The RMA uses the Catalogi API to provide filter options based on "zaaktype".
+
+Add a service of the type ``ZTC``, and make sure to fill out:
+
+- ``API root URL``: the API base URL of the service.
+- ``Extra configuration``: a JSON object with the main catalogue ID, e.g.:
+
+  .. code-block:: json
+
+    {"main_catalogus_uuid": "09a4ae7a-98a3-4178-9559-b22b76cad3db"}
+
+- ``Client ID``: the client ID for your "application" that was registered with the
+  Catalogi API-serving application.
+- ``Secret``: the Secret for your "application" that was registered with the
+  Catalogi API-serving application.
+- ``Authorization type``: ZGW-client_id + secret
+- ``OAS``: URL to the API schema, normally this is ``API root URL`` + ``schema/openapi.yaml``.
+- ``NLX url``: optional NLX outway-URL if the service is to be consumed over the NLX
+  network.
+
+Zaken API
+---------
+
+The Zaken API is used to retrieve the zaken matching the archiving terms. They are the
+objects that are eventually destroyed by this application.
+
+Add a service of the type ``ZRC``. The configuration steps for the Catalogi API apply
+here, with the exception of "Extra configuration" - this is not required.
+
+Documenten API
+--------------
+
+Cases ("zaken") almost always have relations to documents. If the case is being
+destroyed, documents related to it (and no other cases) also need to be destroyed. For
+that purpose, the RMA needs access to the Documents API.
+
+Add a service of the type ``DRC``. The configuration steps for the Catalogi API apply
+here, with the exception of "Extra configuration" - this is not required.
+
```

Comment:
Since we have to remove besluit before deleting zaak, I think it's better to also mention "Besluiten API" and its scopes here

Reply:
forgot about that one! I'll add it too